code/game/gamemodes/blob/overmind.dm

```diff
@@ -36,14 +36,17 @@
 	updateallghostimages()
 	..()
 
+/mob/camera/blob/Life()
+	if(!blob_core)	Destroy()

```

Comment:
@Incoming5643 you can't just call Destroy(), it's called by Qdel() you don't call it on your own I JUST CHECK THIS WITH MP.

Tastes, but I enforce it on everywhere else, when ever files are cleaned these lines are corrected to the style I'm suggesting, just do it Incoming.